trl/models/__init__.py

```diff
@@ -15,6 +15,13 @@
 # limitations under the License.
 from .modeling_base import PreTrainedModelWrapper, create_reference_model
 from .modeling_value_head import AutoModelForCausalLMWithValueHead, AutoModelForSeq2SeqLMWithValueHead
+from .modelling_sd_base import (
```

Comment:
After thinking a bit, we should maybe protect that whole import with `is_diffusers_available()` instead of my comment below, what do you think?

Reply:
Yes, that is the way to go here.